cmd/agent_test.go

```
@@ -1011,6 +1015,9 @@ func TestConfigureRunner(t *testing.T) {
 		if got := allowedVersions.Fields["wget"].GetStringValue(); got != "1.20.3" {
 			t.Fatalf("Configure policy_data allowed_versions.wget = %q, expected %q", got, "1.20.3")
 		}
+		if got := testRunner.configureRequest.PolicyBehavior["policy-bundle"].Values; !reflect.DeepEqual(got, []string{"vpc", "sg"}) {
```

golangci-lint (2.12.2)

[error] avoid direct access to proto field testRunner.configureRequest.PolicyBehavior["policy-bundle"].Values, use testRunner.configureRequest.GetPolicyBehavior()["policy-bundle"].GetValues() instead

(protogetter)